test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -144,20 +212,23 @@ upgrade_downgrade()
         echo "Upgrade/downgrade testing with \"$1\" and \"$2\""
         echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
 
+        file_name_1=$(get_config_file_name $1)
+        file_name_2=$(get_config_file_name $2)
+
         # Alternate running each branch format test program on the second branch's build.
         # Loop twice, that is, run format twice using each branch.
         top="$PWD"
         for am in $3; do
             for reps in {1..2}; do
                 echo "$1 format running on $2 access method $am..."
                 cd "$top/$1/test/format"
-                flags="-1qR $(bflag $1)"
-                ./t $flags -h "$top/$2/test/format/RUNDIR.$am" timer=2
+                flags="-1q $(bflag $1)"
```

Comment:
Removed "reopen" option because I am passing the config file to the test-format that it understands and -c is not compatible with -R.

Reply:
I'm confused, won't this mean we remove the existing database and re-create a new one? Doesn't that mean we've removed the point of this test, that is, to run a single database under multiple releases?

Reply:
The main reason for removing the "-R" option (reopen the existing database) because I want to pass the CONFIG to the test-format binary that it understands. For eg - While doing an upgrade/downgrade test 4.2 test-format binary is run on a 4.4 CONFIG which 4.2 does not understand(it has some configurations which is not supported by older release).
@keithbostic  Please suggest if there is any other better option to handle these scenarios.

Reply:
I have not looked at this, but from my recollections, I suspect that `-R` has to co-exist with `-c`.

Reply:
@keithbostic I created a ticket WT-7721 to update test-format binary to reopen an existing database with different CONFIG, to `-c` and `-R` work together. I have also requested a backport of these changes to the 5.0, 4.4, and 4.2 branches. 

I will keep this ticket WT-7599 on hold until I merge WT-7721.